themes/default/content/docs/intro/cloud-providers/gcp-native/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: GCP-Native
+meta_desc: This page provides on an overview of the Native Google Cloud (GCP-Native) provider for Pulumi.
```

Comment:
```suggestion
meta_desc: This page provides on an overview of the native Google Cloud (GCP-Native) provider for Pulumi.
```

I think that's how we write "native Azure provider"

Reply:
Yep, correct

Reply:
```suggestion
meta_desc: This page provides on an overview of the native Google Cloud (Google-Native) provider for Pulumi.
```

Reply:
fixed.